### DISCUSSION TITLE
router bug ? (router-link-active)

### DISCUSSION THREAD

**Discussion by Vladoker** [nuxt/nuxt#15888] [Questions] [answered]
### I cannot activate the parent element when I am on the child (router-link-active)
nuxt 3 
![image](https://user-images.githubusercontent.com/45948200/213414208-c7af0362-79de-4254-a526-54c96d67201d.png)
![image](https://user-images.githubusercontent.com/45948200/213413996-afd62b48-12ef-40da-9499-6d02c9bff0df.png)
![image](https://user-images.githubusercontent.com/45948200/213414556-31e8827b-8fae-4207-8861-32b93d00bfb9.png)
![image](https://user-images.githubusercontent.com/45948200/213414350-224a01ae-46aa-4a9e-b91b-40a563b3e168.png)

NUXT 2
![image](https://user-images.githubusercontent.com/45948200/213415739-24996231-67f4-44bd-b089-7af74e558467.png)
![image](https://user-images.githubusercontent.com/45948200/213415824-3fb32d44-9d81-453b-912a-488808493dec.png)
![image](https://user-images.githubusercontent.com/45948200/213415988-e0aa87d2-ccac-4697-8ad5-b9518bec474e.png)
![image](https://user-images.githubusercontent.com/45948200/213416088-fad0462a-a004-4585-9fd9-598c7991e2c9.png)

**Comment by ndragun92** [comment, marked as answer]
Hello well, there is some discussion over that. As a workaround, u can use this now

https://stackblitz.com/edit/github-tka8tc?file=layouts%2Fdefault.vue

If you want to follow more on the discussion please refer to this topic

[Check Discussion](https://github.com/nuxt/framework/discussions/2086#discussioncomment-4575574)
